Engine/Module/ModuleResourceManager.cpp

```diff
@@ -27,15 +27,6 @@ bool ModuleResourceManager::Init()
 update_status ModuleResourceManager::PreUpdate()
 {
 
-	if ((thread_timer->Read() - last_imported_time) >= importer_interval_millis)
-	{
-		//importing_thread.join();
-		//importing_thread = std::thread(&ModuleResourceManager::StartThread, this);
-		auto it = std::remove_if(resource_cache.begin(), resource_cache.end(), [](const std::shared_ptr<Resource> & resource) {		
-			return resource.use_count() == 1;
-		});
-		resource_cache.erase(it,resource_cache.end());
-	}
```

Comment:
this created the bug?

Reply:
yes! because prefabs couldn't be found on the cache so were reinstanciated all the time

Reply:
don't remove it. Thi is alreayd fixed in develop, you don't need to remove this.